recipe/meta.yaml

```
@@ -1,47 +1,56 @@
 {% set name = "pywinrm" %}
-{% set version = "0.4.1" %}
-{% set sha256 = "4ede5c6c85b53780ad0dbf9abef2fa2ea58f44c82256a84a63eae5f1205cea81" %}
+{% set version = "0.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5428eb1e494af7954546cd4ff15c9ef1a30a75e05b25a39fd606cef22201e9f1
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-  build:
-    - python
+  host:
     - pip
+    - wheel
+    - python
+    - setuptools
```

```suggestion
    - setuptools >=61.0.0
```

https://github.com/diyan/pywinrm/blob/v0.5.0/pyproject.toml#L3C17-L3C26